Assets/SEE/Game/Operator/EdgeOperator.cs

```diff
@@ -149,14 +149,13 @@ public IOperationCallback<Action> ShowOrHide(bool show, EdgeAnimationKind animat
         /// <param name="enable">Enable or disable animation.</param>
         public void AnimateDataFlow(bool enable = true)
         {
-            if (enable)
+            if (gameObject.TryGetComponent<MeshRenderer>(out MeshRenderer meshRenderer))
             {
-                gameObject.AddOrGetComponent<EdgeDirectionVisualizer>();
+                meshRenderer.material.SetFloat("_EdgeFlowEnabled", enable ? 1.0f : 0.0f);
```

Comment:
We should avoid string-based property lookups for shaders/materials for [performance reasons](https://github.com/JetBrains/resharper-unity/wiki/Avoid-using-string-based-names-for-setting-and-getting-properties-on-Animators,-Shaders-and-Materials). I can't add this as a suggestion, but the fix is to add a new field with the cached property ID which we then reference here:

```csharp
private static readonly int EdgeFlowEnabledProperty = Shader.PropertyToID("_EdgeFlowEnabled");
// [...]
meshRenderer.material.SetFloat(EdgeFlowEnabledProperty, enable ? 1.0f : 0.0f);
```